### ISSUE TITLE
New release (v2.2.1)

### ISSUE THREAD

**Issue by FlipEverything** [spotorm/spot2#274]
Hey @vlucas!
I think we should release a new version as we discussed it previously. 
What do you think?

![image](https://user-images.githubusercontent.com/2407139/44512034-9a23d400-a6b9-11e8-9b38-a949b4fc47ce.png)

**Comment by FlipEverything** [comment]
Should I publish the new release?

**Comment by dertin** [comment]
I would like to see the new version, at the moment I am using the dev-master version from composer.

**Comment by vlucas** [comment]
Yes I agree. Publish away, please! Do you have access to composer to do that as well?

**Comment by FlipEverything** [comment]
[Released v2.2.1](https://github.com/spotorm/spot2/releases/tag/v2.2.1)
No, I don't have access to composer.

**Comment by danopz** [comment]
As I read "Major Changes" and "Add" in the changelog shouldn't it be 2.3 already? New features are minor releases.
> In the next major release (v2.3)...
2.3 is called minor and 3.0 major. :grinning:

**Comment by FlipEverything** [comment]
Thanks for the input. I followed the previous versioning convention, but you have a point at the v2.3 argument. (Updated the description btw.) I don't think that this release should be 2.3 or 3.0. It's just a bunch of small modifications and bugfixes.
